docs/developers/sc-calls-format.md

```diff
@@ -0,0 +1,344 @@
+
+For programmatically created transactions, arguments can be encoded by using one of our SDKs (`erdpy`, `erdjs`, `erdjava`, and so on) or by using built-in components or other libraries
+of the language the transaction is created in. 
+
+There are 2 ways of formatting the data field:
+
+- manually convert each argument, and then join the function name, alongside the argument via the `@` character.
+- use a pre-defined arguments serializer, such as [the one found in erdjs](https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/smartcontracts/argSerializer.ts).
+
+## Converting bech32 addresses (erd1)
+
+Elrond uses `bech32` addresses with the HRP `erd`. Therefore, an address would look like:
+
+`erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th`
+
+:::warning
+Converting a bech32 address into hexadecimal encoding *is not* a simple `string to hex` operation, but requires specialized
+tools or helpers.
+:::
+
+There are many smart contract calls (or built-in function calls) that receive an address as one of their arguments. Obviously, 
+they have to be hexadecimal encoded. 
+
+### Examples
+
+bech32 --> hex
+```
+erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th 
+--> 
+0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1
+
+erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r
+-->
+c70cf50b238372fffaf7b7c5723b06b57859d424a2da621bcc1b2f317543aa36
+```
+
+### Converting addresses using online tools
+
+There are multiple (*unofficial*) tools that one can use in order to convert an address into hexadecimal encoding:
+
+- https://slowli.github.io/bech32-buffer/ (go to `Data`, select `erd` as Tag and `Bech32` as Encoding)
+
+- http://207.244.241.38/elrond-converters/#bech32-to-hex
+
+### Converting addresses using erdpy
+
+Make sure you have `erdpy` [installed](/sdk-and-tools/erdpy/installing-erdpy/).
+
+```
+erdpy wallet bech32 --decode erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th
+```
+
+will output `0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1`.
+
+Alternatively, hex addresses can be converted to bech32 as follows:
+
+```
+erdpy wallet bech32 --encode 0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1
+```
+
+will output `erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th`.
+
+### Converting addresses using erdjs
+
+Find more about `erdjs` [here](/sdk-and-tools/erdjs/erdjs/).
+
+```
+import { Address } from "./address";
+...
+
+const decodedAddress = Address.fromBech32("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th");
+console.log(decodedAddress.hex());
+```
+
+will output `0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1`.
+
+Alternatively, hex addresses can be converted to bech32 as follows:
+
+```
+import { Address } from "./address";
+...
+
+const decodedAddress = Address.fromHex("0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1");
```

Comment:
Can be `address` instead of `decodedAddress`.

Reply:
done